applications/StructuralMechanicsApplication/tests/cpp_tests/test_timoshenko_beams.cpp

```diff
@@ -32,20 +33,22 @@ void CreateBeamModel2N(std::string TimoshenkoBeamElementName)
         auto p_elem_prop = r_model_part.CreateNewProperties(0);
         constexpr auto youngs_modulus = 2.0e+06;
         p_elem_prop->SetValue(YOUNG_MODULUS, youngs_modulus);
-        constexpr auto cross_area = 1.0;
-        p_elem_prop->SetValue(CROSS_AREA, cross_area);
-        constexpr auto i33 = 1.0;
-        p_elem_prop->SetValue(I33, i33);
-        constexpr auto area_effective_y = 1.0;
-        p_elem_prop->SetValue(AREA_EFFECTIVE_Y, area_effective_y);
+        p_elem_prop->SetValue(CROSS_AREA, 1.0);
+        p_elem_prop->SetValue(I33, 1.0);
+        p_elem_prop->SetValue(AREA_EFFECTIVE_Y, 1.0);
 
         const auto &r_clone_cl = KratosComponents<ConstitutiveLaw>::Get("TimoshenkoBeamElasticConstitutiveLaw");
         p_elem_prop->SetValue(CONSTITUTIVE_LAW, r_clone_cl.Clone());
 
         // Create the test element
         constexpr double directional_length = 2.0;
-        auto p_node_1 = r_model_part.CreateNewNode(1, 0.0, 0.0, 0.0);
-        auto p_node_2 = r_model_part.CreateNewNode(2, directional_length, directional_length, directional_length);
+        r_model_part.CreateNewNode(1, 0.0, 0.0, 0.0);
+        r_model_part.CreateNewNode(2, directional_length, directional_length, directional_length);
+        std::vector<ModelPart::IndexType> element_nodes{1,2};
+        if(TNNodes==3) {
```

Comment:
If I'm not mistaken, this can be constexpr if

Reply:
Yes, thank you for the hint.